pkg/controllers/grafana-operator/controller.go

```diff
@@ -46,7 +47,7 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
-	controllerruntime "sigs.k8s.io/controller-runtime"
```

Comment:
Personally, I would prefer to have this as it was before. `ctrl` is a bit ambiguous since it could also mean controller.

Also renaming it in this PR introduces a lot of unrelated changes and makes it a bit harder to review it.

Reply:
I found it confusing the other way .. since monitoring-stack/controller.go calls it `ctrl` and also align with the usage I have seen elsewhere e.g. - https://github.com/kubernetes-sigs/controller-runtime/blob/master/examples/scratch-env/main.go#L26

Reply:
Ok, let's stick to this convention then.